extensions/amazon-lambda-alexa/runtime/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-amazon-lambda-alexa-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>quarkus-amazon-lambda-alexa</artifactId>
+    <name>Quarkus - Amazon Lambda Alexa - Runtime</name>
+    <description>Write AWS Lambda functions for Amazon Alexa</description>
```

Comment:
Is it specific to Amazon Lambda or could be used to use Alexa in another context?

Reply:
If I understand the question correctly, no Amazon Alexa would only be used with AWS Lambda in this context.  Alexa is driven by custom AWS Lambda code "Skills" that give it functionality (for voice, video et al).

Reply:
Oh I should add yes, you can also write Alexa Skills outside of AWS Lambda, any HTTPS interface actually, so containers would be applicable also, I guess that's what you were asking -- in terms of how to name the extension.

Reply:
Yeah, what I was wondering is if this could be a general purpose Alexa extension that we could use for all usage.

That would mean a few things:
- it wouldn't depend on Amazon Lambda. Does it right now? I saw the dependency but I'm not sure it's strictly needed.
- we would add the Alexa dependency to the runtime artifact so that just adding it drags the Alexa artifact.
- obviously, we would change the name to be more generic :).

Reply:
Note that these are suggestions. I'm not accustomed to the Amazon ecosystem so feel free to push back if these are bad ideas.

Reply:
Renamed it to amazon-alexa, makes sense, no need to couple it with the Amazon Lambda extension.